backend/src/main/java/com/redhat/cloud/notifications/processors/camel/CamelTypeProcessor.java

```diff
@@ -41,8 +41,12 @@
 public class CamelTypeProcessor implements EndpointTypeProcessor {
 
     public static final String TOCAMEL_CHANNEL = "toCamel";
-
-    private static final String TOKEN_HEADER = "X-Insight-Token";
+    public static final String PROCESSED_COUNTER_NAME = "processor.camel.processed";
+    public static final String TOKEN_HEADER = "X-Insight-Token";
+    public static final String NOTIF_METADATA_KEY = "notif-metadata";
+    public static final String CLOUD_EVENT_EXTENSION_KEY = "rh-account";
```

Comment:
I think this is no good constant name, as CEs can have multiple extensions.

Reply:
Ok I'll rename it to `CLOUD_EVENT_ACCOUNT_EXTENSION_KEY` in the PR that will test `FromCamelHistoryFiller`.